docs/using-radicle/pushing-and-pulling-changes.md

```diff
@@ -23,44 +23,44 @@ It can be pushed to Radicle with:
+To contribute to another user's project, you must first find it on the Radicle network. To find a project, you need its [**projectID**](understanding-radicle/glossary.md/#project-id). This can be found in the project page under the [project name](understanding-radicle/glossary.md/#project-name), or in a seed node UI.
 
 [insert picture of projectID in seednode UI and in project page]
 
-Once found, copy the projectID into the Upstream search bar. If the project exists in your network of connected peers, it will appear. Following the project will replicate it to you local machine.
+Once found, copy the projectID into the Upstream search bar. Following the project will replicate it to your local machine once it's found in your network of connected peers.
 
 [insert picture of projectID & search bar]
 
 ### Cloning projects
 
-To clone a working copy of a followed project to your computer, you can use the [**Checkout**](understanding-radicle/glossary.md/#checkout) option within the Upstream client. This will clone the project to a specified location on your local machine and initialize a Git repository.
+To clone a working copy of a followed project to your computer, you can use the [**Checkout**](understanding-radicle/glossary.md/#checkout) option within the Upstream client. This will clone the project to a specified location on your local machine.
 
-Commiting and publishing changes to this working copy will create a new [remote](understanding-radicle/glossary.md/#remote) for that project that is uniquely linked to your peerID. You can view this remote in the peer switcher.
+Commiting and publishing changes to this working copy will create a new [remote](understanding-radicle/glossary.md/#remote) for that project which is uniquely linked to your peerID. You can view this remote in the peer switcher.
 
 [insert picture of peer switcher]
 
-Once a remote is created for a project, it will also add the project to your **My Projects** tab. 
+Once a remote is created for a project, it will also move the project to your **My Projects** tab.
 
-## Getting changes from a project 
+## Getting changes from a project
 
 ### Adding remotes
-To view and pull changes from another user's version of your project, you must first **add their remote**. To do this, you must add their **peerID** to your list of remotes. 
+To view and pull changes from another user's version of your project, you must first **add their remote**. To do this, you must add their **peerID** to your list of remotes.
 
 [insert gif of adding peerID to manage remotes modal]
 
 Adding a user's remote is basically adding their version of your project to your machine. Once added, you'll be able to fetch changes from their version and apply them to your working copy. Read more about [how remotes work on Radicle](understanding-radicle/faq.md).
```

Comment:
This isn't  _quite_ right. There must be a path for you to get that remote's changes. This can either be:
* A peer you're connected to has their changes and we fetch those changes from them
* You hear a gossip message about the peer you just tracked

This explanation makes it seem like this happens instantaneously.

Reply:
Yes, I haven't edited this part since our conversation, but will be sure to fix it up in my next iteration.

Reply:
I rewrote as such. Having trouble understanding how "hearing a gossip message" is different from fetching changes from a connected peer. How would you rewrite these sentences? 

_"Adding and following a user's remote is basically adding their version of your project to your machine. To get changes from a new remote you must fetch them from a connected peer that has replicated the remote's changes. Once fetched, you'll be able to pull changes from their version and apply them to your working copy."_

Reply:
I feel like we should stay away from using "version" as it's not really a known term when it comes to git repositories. I think saying something like *... is basically adding their branches of your project to your machine* might be clearer. Otherwise we can use "fork" like github, though I think that's a bit of a mis-use of the term.

Reply:
What about their view?

Reply:
_"Adding and following a user's remote is basically adding their branches of your project to your machine. To get changes from a new remote you must fetch them from a connected peer that has replicated the remote's changes. Once fetched, you'll be able to pull changes from their version and apply them to your working copy."_ 

?

Reply:
I made a suggestion here https://github.com/radicle-dev/radicle-docs/pull/16/files#r520523526. I _think_ it's related but now I'm finding it hard to tell with diffing